treeherder/perf/stats.py

```diff
@@ -270,8 +270,8 @@ def interpret_mann_whitneyu(base, new, pvalue_threshold=PVALUE_THRESHOLD):
     if len(base) < 1 or len(new) < 1:
         return None, None, 0, False
     mann_stat, mann_pvalue = mannwhitneyu(base, new, alternative="two-sided")
-    mann_stat = float(mann_stat) if mann_stat else None
-    mann_pvalue = float(mann_pvalue) if mann_pvalue else None
+    mann_stat = float(mann_stat) if mann_stat else 0
```

Comment:
This needs to be `mann_stat is not None` since the current change would convert a legitimate `None` result to a 0 which we don't want. The same is true for the `mann_pvalue` check below. Also, leave the else part as `else None`